client/src/components/Page/RunsDonePage.js

```diff
@@ -21,34 +22,32 @@ const ScrollContainer = styled.div`
   }
 `;
 export default function RunsDonePage() {
+  const [runs, setRuns] = useState(null);
+  useEffect(() => {
+    async function fetchData() {
+      const newRuns = await getRuns();
+      setRuns(newRuns);
+    }
+    fetchData();
+  }, []);
   return (
     <>
       <RunsDoneContainer>
         <AppHeader title={"Your Runs"} />
         <ScrollContainer>
-          <Link to="/created_run">
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-              isFavorite={true}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
-          </Link>
+          {runs?.map((run) => (
+            <Link key={run.id} to={`/runs/${run.id}`}>
+              <RunCard
+                date={run.date}
+                time={run.time}
+                distance={run.distance}
+                isFavorite={true}
+                runName={run.runName}
```

Comment:
```suggestion
                isFavorite={true}
                {...run}
```

and again a simplification 😃

Reply:
Thanks 👍